### PR TITLE
#435 Create JaversSpringDataAuditable with own OnSaveAuditChangeHandler

### PR DESCRIPTION
**Pull request by arnzel**

### CODE REVIEW

File: javers-spring/src/main/java/org/javers/spring/auditable/aspect/springdata/JaversSpringDataAuditableRepositoryAspect.java [hunk 1]
```diff
@@ -27,6 +27,11 @@ public JaversSpringDataAuditableRepositoryAspect(Javers javers, AuthorProvider a
         this(new OnSaveAuditChangeHandler(javers, authorProvider, commitPropertiesProvider),
                 new OnDeleteAuditChangeHandler(javers, authorProvider, commitPropertiesProvider));
     }
+
```
Comment (bartoszwalacik) [inline javers-spring/src/main/java/org/javers/spring/auditable/aspect/springdata/JaversSpringDataAuditableRepositoryAspect.java]: why another constructor?

**Comment by arnzel** [comment]
As i described in #435 i would like to use an own instance of OnSaveAuditChangeHandler which saves changes only when a given configuration property is set

**Comment by bartoszwalacik** [comment]
ok, but it should be implemented in our spring-boot starters `JaversSqlAutoConfiguration` and `JaversMongoAutoConfiguration` using `@ConditionalOnProperty`

**Comment by arnzel** [comment]
okay you mean adding this annotation to "JaversSpringDataAuditableRepositoryAspect" ?

**Comment by bartoszwalacik** [comment]
No, I mean adding a new spring property and disable aspect when this property is false

**Comment by bartoszwalacik** [comment]
implemented by core team
see http://javers.org/release-notes/

**Pull request closed without merging**